src/styles/index.scss

```diff
@@ -1,3 +1,62 @@
+$days-of-week: (mon, tue, wed, thu, fri, sat, sun);
+
+* {
+  box-sizing: border-box;
+}
+
 body {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
   margin: 0;
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+}
+
+.calendar__day {
+  background-color: #eee;
+  border: 1px solid #000;
+  justify-content: center;
+  align-items: center;
+  width: 100px;
+  height: 100px;
+  transition:
+    background-color 0.5s,
+    transform 0.5s;
+  display: flex;
+
+  &:hover {
+    cursor: pointer;
+    background-color: #ffbfcb;
+    transform: translateY(-20px);
+  }
+
+  @for $i from 1 through 31 {
+    &:nth-child(#{$i}):before {
+      content: '#{$i}';
+    }
+  }
```

Comment:
The loop inside the `.calendar__day` selector is correct, but it's a good practice to place such loops at the end of the selector block to improve readability.